### DISCUSSION TITLE
obconf-qt fully deprecated now?

### DISCUSSION THREAD

**Discussion by 0323pin** [lxqt/lxqt#2115] [Q&A] [answered]
Hi,
NetBSD package maintainer here. I have started to update LXQt to 1.0.0 and I have a short question.

When you guys released 0.17, `obconf-qt` got bumped to 0.16.1. Both we, and Void packaged this as part of 0.17.

Now, with 1.0.0 there was no bump. I've seen the discussion elsewhere that `obconf-qt` is no longer a part of LXQt.

Should I remove it from the package? What's your position as upstream on the subject?
I know you can still use `openbox` with it's own config file but, it might confuse some users. Hence, the question. I'll go with whatever you guys judge correct, as I like the package to be as close as possible to what's intended by upstream.

**Comment by stefonarch** [comment, marked as answer]
If I recall well we just accept and still merge new translations, but do not develop it actively anymore.
So maybe an 0.17.2 could be released @tsujan ?

**Reply by tsujan** [reply]
> What's your position as upstream on the subject?

It has been explained in other places but here may be the best place to put it:

 * LXQt has removed all traces of treating an X11 window manager or compositor in special ways. All DE-agnostic X11 WMs can be used with LXQt and distros are free to choose one as their default (Openbox, xfwm4, KWin,...).
 * Openbox isn't developed anymore, although it's still usable.

> So maybe an 0.17.2 could be released @tsujan ?

I think you meant 0.16.2. The code hasn't changed since 0.15.0. IMHO, its repository can be archived. If LXQt team agrees on that, 0.16.2 could be released as the last version only because of translations.

**Reply by 0323pin** [reply]
Honestly, I've seen most of the discussions regarding this and want to remove it from the package, just wanted to make sure you were on the same line.

I'll remove it from the package.

**Reply by stefonarch** [reply]
> I think you meant 0.16.2. The code hasn't changed since 0.15.0. IMHO, its repository can be archived. If LXQt team agrees on that, 0.16.2 could be released as the last version only because of translations.

Sure, I meant release 16.0.2,  and remove it from weblate. 
Translations are complete except swedish and estonian (this translator is quite active, I could ping him) https://github.com/lxqt/obconf-qt#translations

**Reply by Chiitoo** [reply]
Hmmm, Finnish translation missing completely from there.

Could probably squeeze in a quick one before the archival, but I'm not sure it would be worth the efforts.

**Reply by Chiitoo** [reply]
It looks like I don't have the permission to start a new translation, which makes sense (are they added via Weblate or a PR?).

If it seems like a good idea, and someone with the power to do so starts one up for Finnish, I'll do the translation work.

**Comment by stefonarch** [comment]
You needed upgraded permissions, you should be able to add finnish now.

**Reply by Chiitoo** [reply]
Yerp, thanks!
